name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/BarclaysPDFExtractor.java

```diff
@@ -0,0 +1,68 @@
+package name.abuchen.portfolio.datatransfer.pdf;
+
+import static name.abuchen.portfolio.util.TextUtil.trim;
+
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.Block;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.DocumentType;
+import name.abuchen.portfolio.datatransfer.pdf.PDFParser.Transaction;
+import name.abuchen.portfolio.model.AccountTransaction;
+import name.abuchen.portfolio.model.Client;
+
+
+@SuppressWarnings("nls")
+public class BarclaysPDFExtractor extends AbstractPDFExtractor
+{
+    public BarclaysPDFExtractor(Client client)
+    {
+        super(client);
+
+        addBankIdentifier("Barclays");
+        addBankIdentifier("Barclays Bank Ireland PLC");
+        addBankIdentifier("22792 Hamburg");
```

Comment:
```suggestion
        addBankIdentifier("USt-IdNr.: DE 319 453 063");
        addBankIdentifier("Handelsregister HRB: 153530");
        addBankIdentifier("barclays.de");
```
If someone lives in this area with their address (zip code + city), then we have a problem.
Maybe "_barclays.de_" or "_Handelsregister HRB: 153530_" or "_USt-IdNr.: DE 319 453 063_" ?